lib/openhab/dsl/items/color_item.rb

```diff
@@ -48,10 +48,51 @@ def coerce(other)
         # convert it to an HSBType
         # @!visibility private
         def format_type(command)
+          return format_hash(command.to_h) if command.respond_to?(:to_h)
```

Comment:
I'd recommend using to_hash here, not to_h. It will avoid round-tripping to a hash-and-back if you pass in an HSBType (which now responds to to_h).